ci/build.yml

```diff
@@ -105,6 +105,11 @@ jobs:
       inputs:
         pathtoPublish: 'bazel-bin/docs/html.tar.gz'
         artifactName: 'Docs bundle'
+    - task: PublishBuildArtifacts@1
+      displayName: 'Publish test evidence'
+      inputs:
+        pathtoPublish: 'bazel-bin/ledger/test-evidence/daml-test-evidence.tar.gz'
+        artifactName: 'Test Evidence'
```

Comment:
What are you trying to achieve here? This is not required to publish the artifacts to the assembly repo.

Reply:
Actually this will break our ability to make pre-2.2 patch releases, so I'd prefer it be left out.

Reply:
as agreed with @mziolekda we would like test evidence also being published during every PR build so it can be reviewed

Reply:
Are you ever going to actually look at it?

Regardless, as it stands, this is breaking our release process. If you really want to have this, please hide it behind a flag. I recommend removing this from this PR and opening a separate one for that as this one is large enough as it is.

You can add a `condition` to this step based on the value of a variable emitted in a previous Bash step (which you should add) that tests for the existence of the tarball, then uses the `set_var` function from the bash-lib. Ping me on Slack if you need more step-by-step instructions.

Reply:
I have removed this change out of this PR